ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -28,6 +28,13 @@
 - name: start invoker using docker cli
   shell: >
         docker run -d
+        --pid=host
+        --userns=host
+        --privileged
+        -v /sys/fs/cgroup:/sys/fs/cgroup
+        -v /run/runc:/run/runc
+        -v /usr/bin/docker-runc:/usr/bin/docker-runc
+        -v /usr/lib/x86_64-linux-gnu/libapparmor.so.1:/usr/lib/x86_64-linux-gnu/libapparmor.so.1
```

Comment:
most likely not. apparmor is ubuntu-specific and not a given on all *nix systems.
this is also part of the reason why apparmor is not yet merged. my suggestion would be to provide something like a "feature discovery", maybe as a singleton like whiskconfig. it should be able to discover available docker features based on the `docker info` command (or corresponding rest api call)